lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -2172,6 +2169,7 @@ namespace Js
         if (state->GetInternalProperty(state, Js::InternalPropertyIds::HiddenObject, &hiddenObject, nullptr, scriptContext))
         {
             dtf = reinterpret_cast<FinalizableUDateFormat *>(hiddenObject);
+            INTL_TRACE("Using previously cached UDateFormat");
```

Comment:
Looks like the xplat builds don't like this macro not having additional arguments?